recipes/diffprivlib/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "diffprivlib" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8dba0a65ecd52c79b71d8f3837228386a154df7039fa149eb06bc48b707b9bfa
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

The parameters are no longer needed.
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```